### PR TITLE
chore(docs): revert removal of custom search scoring

### PR DESCRIPTION
**Pull request by BobDotCom**
## Summary
This pull request reverts the removal of a custom documentation search scoring script. This script was removed when the docs were refactored, and it seems to have caused issues with search priorities. 

## Information

- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [x] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [ ] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [ ] I have updated the changelog to include these changes.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -380,7 +380,7 @@ def write_new():
 
 # The name of a javascript file (relative to the configuration directory) that
 # implements a search results scorer. If empty, the default will be used.
-# html_search_scorer = "_static/scorer.js"
+html_search_scorer = "_static/scorer.js"
```
Comment (plun1331) [inline docs/conf.py:383]:
```suggestion
html_search_scorer = "_static/js/scorer.js"
```

**Review by plun1331 — CHANGES_REQUESTED** [review]
Incorrect path

**Review by Lulalaby — APPROVED** [review]

**Comment by Lulalaby** [comment]
Before:
before

After:
after

**Comment by BobDotCom** [comment]
Here's a comparison when searching for "Sticker"
| Prior to Refactor | Current | This PR |
|--------|--------|--------|
| Screenshot 2023-04-17 at 10 42 53 AM | image | Screenshot 2023-04-17 at 10 43 57 AM |

**Review by plun1331 — APPROVED** [review]

**Review by ChickenDevs — APPROVED** [review]

**Pull request merged**